### ISSUE TITLE
Gurobi with R version 3.5.0

### ISSUE THREAD

**Issue by jeffreyhanson** [prioritizr/prioritizr#88]
The latest version of the _gurobi R_ package (version 8.0.0) is not compatible with the latest version of _R_ (version 3.5.0). Unfortunately, the current solution to this problem seems to require downgrading _R_ (version 3.4.4). For information on this issue, please see this post on the Gurobi emailing list: https://groups.google.com/forum/#!topic/gurobi/D_A81BvJupY

**Comment by ricschuster** [comment]
Do you think we should wait to release the latest version of `prioritzr` to CRAN in this case? Too bad that nobody from Gurobi has replied to the Google Group post yet.

**Comment by jeffreyhanson** [comment]
I don't think there's anything we can do to fix it so I don't know if it's worth waiting to release it? The only issue I can think of is it might be annoying for Windows users if the win-builder binaries are built for 3.5.0 and they aren't compatible with 3.4.4, so Windows users might need to manually install the Github version of _prioritizr_ from source to use it with 3.4.4?

**Comment by jeffreyhanson** [comment]
Thanks, yeah that's a good idea. I'll test it out on my windows computer when the compiled windows  version is available and let you know how it goes.

**Comment by javierfajnolla** [comment]
I have been able to install gurobi's R package by building it from source. It is not optimal... but just in case. In the technical support mail, they showed me the source code at:
http://files.gurobi.com/gurobi800/gurobiR.tar.gz

**Comment by jeffreyhanson** [comment]
Awesome! Thank you so much @javierfajnolla for sharing this! I'll see if I can put together an R script which automates the install process.

**Comment by jeffreyhanson** [comment]
Gurobi has appears to have fixed this with version 8.0.1.

```
> library(gurobi)
Loading required package: slam

Warning message:
package ‘gurobi’ was built under R version 3.5.0 
> 
> packageVersion("gurobi")
[1] ‘8.0.1’
```
